R/ecopy.R

```diff
@@ -36,7 +38,7 @@ ecopy <- function(
     if( !is.null( ncol(x) ) ){
       
       utils::write.table( x,
-        file = "clipboard", sep="\t", 
+        file = paste0("clipboard-", buffer, ""), sep="\t", 
```

Comment:
you don't need the extra "" but it also won't break anything - please fix it anyway though.